.github/workflows/publish.yml

```diff
@@ -0,0 +1,30 @@
+name: publish
+description:
+
+on:
+  workflow_dispatch:
+    tags:
+      - 'v*'
+
+jobs:
+  publish:
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3.5.2
+        with:
+          ref: ${{ github.ref }}
+
+      - name: Use node@16
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.20.0
+
+      - name: Authenticate NPM
+        run: echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
+
+      - name: Publish
+        run: |
+          yarn publish
```

Comment:
For a release candidate, will this use the `next` or `beta` flag (rather than `latest`)? 

If it does use `latest`, it may cause `npm install` to cause users to upgrade to a beta version:

package.json
```
"openzeppelin/defender-admin-client": "^1.44.0",
```

may use `1.45.0-rc.1` if that were tagged `latest`. It's possible the `-beta` overrides this - not sure - but it would be worth checking.

Reply:
All the publishes uses latest since we are not explicitly specifying `--dist-tag` with `lerna publish`